solidity/contracts/BondedECDSAKeepFactory.sol

```diff
@@ -405,16 +417,39 @@ contract BondedECDSAKeepFactory is IBondedECDSAKeepFactory, CloneFactory {
     /// @param _operator operator's address
     /// @return True if has enough active stake to participate in the network,
     /// false otherwise.
-    function hasMinimumStake(address _operator) public view returns(bool) {
-        return (
-            tokenStaking.activeStake(_operator, address(this)) >= minimumStake
-        );
+    function hasMinimumStake(address _operator) public view returns (bool) {
+        return
+            tokenStaking.activeStake(_operator, address(this)) >= minimumStake;
     }
 
     /// @dev Gets the stake balance of the specified operator.
     /// @param _operator The operator to query the balance of.
     /// @return An uint256 representing the amount staked by the passed operator.
-    function balanceOf(address _operator) public view returns(uint256) {
+    function balanceOf(address _operator) public view returns (uint256) {
         return tokenStaking.balanceOf(_operator);
     }
+
+    /// @notice Slashes keep members' KEEP tokens. For each keep member it slashes
+    /// amount equal to the minimum stake configured for this factory.
+    function slashKeepMembers() public onlyKeep {
+        BondedECDSAKeep keep = BondedECDSAKeep(msg.sender);
+
+        tokenStaking.slash(minimumStake, keep.getMembers());
+    }
+
+    /// @notice Notifies the factory that a keep has been closed.
+    /// @dev Function should be called by a keep which is being closed.
+    function notifyKeepClosed() public onlyKeep {
+        keeps[msg.sender] = false;
```

Comment:
Late hit here, but is this equivalent from a gas perspective to `delete keeps[msg.sender]`? If not, would it make sense to delete rather than set to `false`?

Reply:
It's the same. `delete` basically sets to a default value.